sdk/python/symbolchain/sc/__init__.py

```diff
@@ -635,12 +637,11 @@ def size(self) -> int:
 		return size
 
 	@classmethod
-	def deserialize(cls, payload: bytes | bytearray | memoryview) -> Transaction:
-		buffer = memoryview(payload)
+	def _deserialize(cls, buffer: memoryview, instance) -> (int, int):
+		size_ = len(buffer)
 		size_ = int.from_bytes(buffer[:4], byteorder='little', signed=False)
 		buffer = buffer[4:]
 		buffer = buffer[:size_ - 4]
```

Comment:
Can this be rewritten as ``buffer = buffer[4:size_ - 4]``

Reply:
i can tweak the generator a bit 👍